common/src/main/java/com/wynntils/mc/utils/objects/ServerIcon.java

```diff
@@ -42,7 +42,7 @@ public ServerIcon(ServerData server, boolean allowStale, Consumer<ServerIcon> on
 
         // Try default
         ResourceLocation destination =
-                new ResourceLocation("wynntils/servers/" + Hashing.sha1().hashUnencodedChars(server.ip) + "/icon");
```

Comment:
This seem unrelated..?

Reply:
It's shoehorned in, but mostly a small fix.

Reply:
What is the fix about? The texture does look fine to me?

Reply:
Minecraft's server list stores icons at this resource location, which means that this location should be checked for an already existing icon.

Reply:
But we switched to using our own static icon instead..?

Reply:
We use our static icon if we don't have a cache of it.

Reply:
This is about the original implementation.